tests/e2e/billing/journal/test_sync_journal.py

```
@@ -0,0 +1,108 @@
+    new_billing_journal_request_data = billing_journal_factory(
+        name="E2E Created Billing Journal",
+    )
+
+    created_billing_journal = mpt_vendor.billing.journals.create(new_billing_journal_request_data)
+
+    yield created_billing_journal
+
+    try:
+        mpt_vendor.billing.journals.delete(created_billing_journal.id)
+    except MPTAPIError as error:
+        print(f"TEARDOWN - Unable to delete billing journal: {error.title}")  # noqa: WPS421
```

Ruff (0.14.10)

Unused `noqa` directive (unknown: `WPS421`)

Remove unused `noqa` directive

(RUF100)